js/contributor-cards/excluded-list.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
I think we should also move this file to /data/contributors/

I don't think we need to name this file jakarta-9-excluded-list.json since I would envision us wanting to re-use the same list for future releases!

Reply:
done